unfetter-discover/unfetter-ui
# Issue 1057 (Object assessment creation/management in baselines wizard)

Fixes unfetter-discover/unfetter#1057
Fixes unfetter-discover/unfetter#1058

Object assessments are now created in the capability selector for any capabilities that are selected and not associated with any existing object assessments in the baseline.  Also, in addition to the current capability being set based on the current step in the wizard, the associated object assessment is also set in the baseline store so the capability component can use it to populate the weightings.

## To test:
1. Pull this PR
2. Restart unfetter-ui:  `docker-compose restart unfetter-ui`
3. Proceed into baselines and enter the baselines wizard
4. In the capability selector, verify (using Redux Devtools) that `baselineObjAssessments` in `baselines` is updated after each capability is selected.
5. When entering the capability component (where attack patterns are selected and weightings are entered), verify (again, using Redux Devtools) that the `currentObjectAssessment` is set correctly, and has assessed objects inherited from the corresponding category based on it its capability.
